xml/System.Collections.Generic/HashSet`1.xml

```diff
@@ -127,69 +127,15 @@
-  
- The primary difference between LINQ set operations and <xref:System.Collections.Generic.HashSet%601> operations is that LINQ set operations always return a new <xref:System.Collections.Generic.IEnumerable%601> collection, whereas the <xref:System.Collections.Generic.HashSet%601> equivalent methods modify the current collection.  
-  
- Typically, if you must create a new set or if your application needs access only to the provided set operations, using LINQ set operations on any <xref:System.Collections.Generic.IEnumerable%601> collection or array will be sufficient. However, if your application requires access to additional set operations, or if it is not desirable or necessary to create a new collection, use the <xref:System.Collections.Generic.HashSet%601> class.  
-  
- The following table shows the <xref:System.Collections.Generic.HashSet%601> operations and their equivalent LINQ set operations.  
-  
-|HashSet operation|LINQ equivalent|  
-|-------------------------------|---------------------|  
-|<xref:System.Collections.Generic.HashSet%601.UnionWith%2A>|<xref:System.Linq.Enumerable.Union%2A>|  
-|<xref:System.Collections.Generic.HashSet%601.IntersectWith%2A>|<xref:System.Linq.Enumerable.Intersect%2A>|  
-|<xref:System.Collections.Generic.HashSet%601.ExceptWith%2A>|<xref:System.Linq.Enumerable.Except%2A>|  
-|Not provided.|<xref:System.Linq.Enumerable.Distinct%2A>|  
-|<xref:System.Collections.Generic.HashSet%601.SymmetricExceptWith%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.Overlaps%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.IsSubsetOf%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.IsProperSubsetOf%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.IsSupersetOf%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.IsProperSupersetOf%2A>|Not provided.|  
-|<xref:System.Collections.Generic.HashSet%601.SetEquals%2A>|Not provided.|  
-  
-   
-  
-## Examples  
- The following example demonstrates how to merge two disparate sets. This example creates two <xref:System.Collections.Generic.HashSet%601> objects, and populates them with even and odd numbers, respectively. A third <xref:System.Collections.Generic.HashSet%601> object is created from the set that contains the even numbers. The example then calls the <xref:System.Collections.Generic.HashSet%601.UnionWith%2A> method, which adds the odd number set to the third set.  
-  
+    <remarks>For more information about this API, see <see href="/dotnet/fundamentals/runtime-libraries/system-collections-generic-hashset{t}">System.Collections.Generic.HashSet&lt;t&gt; class</see>.</remarks>
```

Comment:
This reads weird, since it's the current page that is supposed to be the doc of the *"`System.Collections.Generic.Hash<T>` class"*. I would rather repharese it as following:

```suggestion
    <remarks>For more information, see the supplemental API remarks on <see href="/dotnet/fundamentals/runtime-libraries/system-collections-generic-hashset{t}">HashSet&lt;T&gt; class</see>.</remarks>
```

Type arguments should be definitely capitalized, and seeing the whole namespace feels unnecessarily verbose here.

The TOC in the linked doc contains an extra `\` behind the class name (`HashSet<T>\ class`):
https://learn.microsoft.com/en-us/dotnet/fundamentals/runtime-libraries/system-collections-generic-hashset%7Bt%7D

Reply:
Thanks - good catch. Fixed the backslash here: https://github.com/dotnet/docs/pull/39107